Decred Wallet/Features/More/MoreMenuViewController.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  MoreViewController.swift
+//  Decred Wallet
+//
+//  Created by Suleiman Abubakar on 26/12/2019.
+//  Copyright © 2019 Decred. All rights reserved.
+//
+
+import UIKit
+
+class MoreMenuViewController: UIViewController,UITableViewDataSource,UITableViewDelegate   {
+    @IBOutlet weak var moreMenuTableView: UITableView!
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        let emptyButton = UIBarButtonItem.init()
+        let barButtonImg = UIBarButtonItem(image: UIImage(named: "ic_decred")?.withRenderingMode(.alwaysOriginal),
+        style: .done, target: self,
+        action: nil)
+        let barButtonTitle = UIBarButtonItem(title: LocalizedStrings.more, style: .plain, target: self, action: nil)
+        barButtonTitle.tintColor = UIColor(hex: "#091440") // UIColor.appColor.darkblue
+        
+        self.navigationItem.leftBarButtonItems = [emptyButton, emptyButton, barButtonImg, barButtonTitle]
```

Comment:
perhaps we could consider using constraints or initializing with a valid `CGPoint` rather than adding empty components as padding?

Reply:
Or use a custom view for navigation bar.